v3/src/components/codap-component.tsx

```diff
@@ -71,14 +72,19 @@ export const CodapComponent = observer(function CodapComponent(props: IProps) {
   if (!info) return null
 
   const { TitleBar, Component, tileEltClass } = info
-  const classes = clsx("codap-component", tileEltClass, { minimized: isMinimized },
-                    { shadowed: uiState.isFocusedTile(tile.id) || uiState.isHoveredTile(tile.id) })
+  const classes = clsx("codap-component", tileEltClass, { focused, minimized: isMinimized })
   return (
     <TileModelContext.Provider value={tile}>
       <TileSelectionContext.Provider value={tileSelection}>
         <CodapComponentContext.Provider value={codapComponentRef}>
-          <div className={classes} ref={codapComponentRef} key={tile.id} data-testid={tileEltClass}
-            onFocus={handleFocusEvent} onPointerDownCapture={handleFocusEvent}>
+          {/* FIXME: Is it ok to remove the onFocus handler and change onPointerDownCapture to onPointerDown? */}
```

Comment:
I think not. This means that clicking on a tile will only focus the tile for tiles that don't swallow the event (which is probably what's happening in the case of the attribute headers in the table). My proposal for this PR would be to put this back the way it was and then make a new story for special-casing the behavior of the minimize button.